test/cqlpy/cassandra_tests/batch_test.py

```diff
@@ -5,7 +5,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-from cassandra_tests.porting import *
+from test.cqlpy.cassandra_tests.porting import *
```

Comment:
I have a serious question:
Doesn't Python have **relative** import paths, so you don't need to repeat "test.cqlpy.cassandra_tests" when you're already in that directory?

Also please confirm that after this change you can do something like
```
cd test/cql-pytest; pytest
```
And it will work (even though we're in a subdirectory, the test/cqlpy/... path works)?